packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -378,10 +378,16 @@ export function ArticleEditor({id}: ArticleEditorProps) {
       }
 
       setChanged(false)
-      Notification.success({
-        title: t('articleEditor.overview.articlePublished'),
-        duration: 2000
-      })
+
+      publishData === new Date() || publishDate < new Date()
```

Comment:
I think you can use smaller or equal for the comparison instead of comparing or smaller. `publishDate <= new Date()`.

Also i think you have a typo. The first argument is publishDat**a** instead of publishDat**e**.